### PR TITLE
charts: set requirements to new versions

### PR DESCRIPTION
**Pull request by rokroskar**

### CODE REVIEW

**Comment by leafty** [comment]
Note: tests not passing is probably due to change of UI

File: charts/renku/requirements.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 - name: renku-notebooks
   alias: notebooks
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.2.1-b0c93bb"
+  version: "0.2.1-3702f3b"
 - name: renku-gateway
   alias: gateway
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.2.0"
+  version: "0.2.0-9094770"
```
Comment (leafty) [inline charts/renku/requirements.yaml]: can we do something about this not being `0.2.1-x`?

Comment (rokroskar) [reply charts/renku/requirements.yaml]: sure, and the UI as well I suppose

Comment (leafty) [reply charts/renku/requirements.yaml]: yup -> https://github.com/SwissDataScienceCenter/renku-ui/pull/344

File: charts/renku/requirements.lock [hunk 2]
```diff
@@ -1,16 +1,16 @@
 dependencies:
 - name: renku-ui
   repository: https://swissdatasciencecenter.github.io/helm-charts/
-  version: 0.2.0
+  version: 0.2.1-9fc6260
 - name: renku-notebooks
   repository: https://swissdatasciencecenter.github.io/helm-charts/
-  version: 0.2.1-b0c93bb
+  version: 0.2.1-3702f3b
```
Comment (leafty) [inline charts/renku/requirements.lock]: wait for update

**Pull request merged**